src/txFactory/identityManagerTransactionFactory.ts

```diff
@@ -342,10 +354,11 @@ export function getEntity(web3, didEntity) {
  */
 export function entitiesList(web3) {
   const transaction = Object.assign({}, config.basicTransaction)
-  transaction.data = web3.eth.abi.encodeFunctionCall(
+  const delegatedData = web3.eth.abi.encodeFunctionCall(
     config.contractsAbi.AlastriaIdentityEntity.entitiesList,
     []
   )
+  transaction.data = delegated(web3, delegatedData)
```

Comment:
In a call function (getters) there is no need to use the proxy. Revert this function.